theme-dev/README.md

```diff
@@ -65,28 +65,30 @@ Used to show name of the theme, author, etc. They are in the "manifest" section
 If done correctly it should look similar to this in the .JSON
 ![image](https://user-images.githubusercontent.com/84905506/132266565-ff27a087-4e36-48ca-baca-2a1d823939fd.png)
 
+
   
 
 ### Background Image Strings
 
-These are the strings for adding a background and also giving it transparency(alpha). They are in the "background" section in the .json
+These are the strings for adding a background and also giving it transparency (alpha). They are in the "background" section of the .json
 
 **Warning: Your Background won't be visible unless you enable transparency!**
 ```json
 {
-  "url": "url for the bg image, recommended and trusted to use cdn.discordapp.com links",
+  "url": "url for the bg image, recommended and trusted to use cdn.discordapp.com links,"
+"(they expire now though, so use something else),"
```

Comment:
This shouldnt be a separate attribute to URL, better to just remove the entire recommended thing and just leave it as "url for the background image"
Just so we can follow the proper JSON structure

Reply:
That would make more sense, yeah